src/can.c

```diff
@@ -586,6 +605,8 @@ int thingset_can_init_inst(struct thingset_can *ts_can, const struct device *can
 #endif
     k_sem_init(&ts_can->request_response.sem, 1, 1);
     k_sem_init(&ts_can->report_tx_sem, 0, 1);
+    k_timer_init(&ts_can->timeout_timer, thingset_can_timeout_timer_expired,
+                 thingset_can_timeout_timer_stopped);
```

Comment:
If the stop function is not needed, just set it to `NULL`:
```suggestion
    k_timer_init(&ts_can->timeout_timer, thingset_can_timeout_timer_expired, NULL);
```